app/config/config.yml

```diff
@@ -184,6 +184,7 @@ sonata_admin:
         groups:
             content:
                 label: Content
+                icon: '<i class="fa  fa-file-text-o"></i>'
```

Comment:
why the old `<i>` tag? would `<span>` not be more typical? and is the extra whitespace after "fa" intentional?